### ISSUE TITLE
Cannot deploy to different region

### ISSUE THREAD

**Issue by tschneid** [zodern/mup-aws-beanstalk#22]
I tried to deploy to `eu-central-1` but got the following error on `mup deploy`:
```
InvalidParameterCombination: Unable to download from S3 location (Bucket: mup-my-app  Key: mup/bundles/my-app/6).
Reason: The bucket is in this region: us-east-1. Please use this region to retry the request
```
In https://github.com/zodern/mup-aws-beanstalk/blob/master/src/aws.js however it seems like the specified region from `mup.js` is respected when creating the bucket. 

My `mup.js`:
```js
module.exports = {
  app: {
    type: 'aws-beanstalk',
    name: process.env.CI_PROJECT_NAME,
    path: './',
    env: {
      ROOT_URL: process.env.ROOT_URL,
      MONGO_URL: process.env.MONGO_URL
    },
    auth: {
      id: process.env.AWS_ACCESS_KEY_ID,
      secret: process.env.AWS_SECRET_ACCESS_KEY
    },
    minInstances: 1,
    maxInstances: 1,
    instanceType: 't2.small',
    region: 'eu-central-1',
    customBeanstalkConfig: [
      {
        namespace: 'aws:elasticbeanstalk:container:nodejs',
        option: 'NodeVersion',
        value: '8.9.3'
      }
    ],
  },
  plugins: ['mup-aws-beanstalk']
};
```

Am I missing something or is this some strange behavior of the AWS SDK?

**Comment by zodern** [comment]
https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#createBucket-property

It needs to provide a LocationConstraint when creating the bucket.

**Comment by zodern** [comment]
This is fixed in 1.5.1.

**Comment by tschneid** [comment]
For the sake of completeness: I had to remove the previously created bucket (in the default region `us-east-1`) using the AWS console (and [wait about an hour](https://stackoverflow.com/a/16553056)). Since this will supposedly happen again when changing the region setting, you might want to document this somewhere (or create buckets with unique names). Aside from that it is working as intended – thanks @lwluc for the fix!
